assets/js/theme/common/product-details.js

```diff
@@ -76,6 +93,19 @@ export default class ProductDetails extends ProductDetailsBase {
         this.previewModal = modalFactory('#previewModal')[0];
     }
 
+    registerAddToCartValidation() {
+        this.addToCartValidator.add([{
+            selector: '[data-quantity-change] > input[id^="qty"]',
```

Comment:
Proposal: use data attribute instead of id regexp here?

Reply:
I switched to `.form-input--incrementTotal` since noticed that it's used further in code.